<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="DatBOI" failures="1" tests="5" name="Test" time="46.69" errors="0" timestamp="2023-08-08T23:40:55 EDT">
  <testcase classname="extentReporterClass.TutorialsNinja_Register" name="RegisterAccountWithValidRequirements" time="2.908"/>
  <testcase classname="extentReporterClass.TutorialsNinjaLoginTest" name="clickOnMyAccount" time="0.157"/>
  <testcase classname="extentReporterClass.TutorialsNinjaLoginTest" name="clickOnLogin" time="0.689"/>
  <testcase classname="extentReporterClass.TutorialsNinjaLoginTest" name="enterValidCredentials" time="1.005">
    <failure type="java.lang.AssertionError" message="null">
      <![CDATA[java.lang.AssertionError: null
at extentReporterClass.TutorialsNinjaLoginTest.enterValidCredentials(TutorialsNinjaLoginTest.java:48)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 29 stack frames]]>
    </failure>
  </testcase> <!-- enterValidCredentials -->
  <testcase classname="extentReporterClass.TutorialsNinjaLoginTest" name="clickOnLoginButton" time="0.0">
    <skipped/>
  </testcase> <!-- clickOnLoginButton -->
</testsuite> <!-- Test -->
